### ISSUE TITLE
RTSP H264 Server closes not enabling

### ISSUE THREAD

**Issue by sanjeewasam** [EliasKotlyar/Xiaomi-Dafang-Hacks#375]
Hi
This is my second time installing the hack. First I got tired of it as i started getting unable to load image and also it is very slow via the http://............. and went back to original Xiaomi firmware (it is not as easy as removing SD card anymore)

After that I got cheap Bulb camera's -6 of them. No hacks and work perfectly as RTSP steam in Blue Iris and other apps. Also foscam

I am keen to give this a go so I did the hack again. downloaded files as of 24/05/18 again and configured. I got the image once in http://   and in blue iris and then is it is gone. Not even in http://<IP> - unable to load image. Why is this happening?. RTSP server H264 or RTSP MJPEG Server in camera controls get disabled no sooner you enable them. I formatted the SD card and copied the files again. No luck. RTSP stream is unavailable.

I think of getting rid of Defang. Is the older Xiaofang hack working better? Defang hack to me  a just time waster at least in my experince. Much appreciate if someone could proove me wrong

**Comment by jmtatsch** [comment]
Have a look at the rtspserver.log. Then help improving this repo.

**Comment by sanjeewasam** [comment]
Thanks. I could not locate the log files but I did the following and it worked. May be that I missed some instructions. However it should go to installation steps or FAQs to avoid people wasting their time the developer's time

In cofig folder I renamed rtspserver.conf.dist as rtspserver.conf. 

I believe all for all services including MQTT one need to make copy of respective xxx.conf.dist and rename it as xxx.conf and then change as required

**Comment by jmtatsch** [comment]
rtspserver.conf is initialised automatically from rtspserver.conf.dist if it doesn't exist. Only exception is mqtt where it doesn't make sense to guess the users mqtt broker.

**Comment by zavviodotcom** [comment]
Thanks for your response. I will try one of the bulb cameras. Camera
recording is a new feature on Zavvio Web Server, which has traditionally
been used to broadcast information via web pages on a local wireless
network that is not connected to Internet. Support is provided for
configuration of 4 cameras, but performance hasn't been characterized for
multiple cameras (if you have RPi 3 or Zero W, you are welcome to try it,
activation has always been free). The purpose of ZWS for camera recording
is different from applications such as Blue Iris; it is currently intended
for loop recording in public places where anyone can retrieve the videos.

When I tested DaFang I noticed that the video files are much larger than
XiaoFang, and will fill a typical SD card quickly, which means that 4
DaFang cameras is less practical. I am hoping to get some time to look at
contributing code to this project for controlling the RTSP server on the
camera to generate smaller video files.
